documentation/modules/rn-2.4.adoc

```diff
@@ -0,0 +1,143 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Release_notes/master.adoc
+
+[id="rn-24_{context}"]
+= {project-full} 2.4
+
+You can migrate virtual machines (VMs) from VMware vSphere or {rhv-full} or {osp} to {virt} with {the-lc} {project-first}.
+
+The release notes describe technical changes, new features and enhancements, and known issues.
+
+[id="technical-changes-24_{context}"]
+== Technical changes
+
+This release has the following technical changes:
+
+.OpenStack migration
```

Comment:
@anarnold97 sorry, I wasn't specific about "section" before - what i meant is to have it as the first item in the  "New features and enhancements" section below

Reply:
@ahadas thanks for all your help. It is tough picking up something like this half way through